data/json/items/comestibles/med.json

```diff
@@ -1288,5 +1288,54 @@
         "addiction_potential" : 40,
         "addiction_type" : "sleeping pill",
         "use_action" : "XANAX"
+    },
+    {
+        "id" : "disinrag",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "disinfectant soaked rag",
+        "name_plural" : "disinfectant soaked rags",
+        "description" : "A rag soaked on disinfectant, maybe not as effective as pouring it on the wound, but useful to optimize the bottle content.",
+        "weight" : 6,
+        "volume" : 1,
+        "charges" : 4,
+        "price" : 900,
+        "stack_size" : 20,
+        "material" : "cotton",
+        "symbol" : "!",
+        "color" : "white",
+        "use_action" : {
+            "type" : "heal",
+            "limb_power" : 10,
+            "head_power" : 10,
+            "torso_power" : 10,
+            "bite" : 0.75,
+            "move_cost" : 100
+        }
+    },
+    {
+        "id" : "disincotton_ball",
+        "type" : "COMESTIBLE",
```

Comment:
Suggest same `"copy-from":` line here from the base item (in this case, the non-soaked cotton balls).